### PR TITLE
created three classes in person.rb

### PR DESCRIPTION
**Pull request by businesskaniki**
added linters
created  person rb and added   person teacher and student classes

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline person.rb]: - [ ] kindly make name age and id accessible. like this ` attr_accessor :name, :age `,  ``attr_reader :id``

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    "¯\(ツ)/¯"
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline person.rb]: - [ ] Kindly create an individual .rb file for this class

File: person.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline person.rb]: - [ ] also kindly create an individual .rb file for this class

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @businesskaniki  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_writer :id
```
Comment (cynthiainga) [inline person.rb]: - [ ] As mentioned by the previous reviewer this is the `attr_reader` and not `attr_writer`, the id should be read and not written 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @businesskaniki, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

 ## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello @businesskaniki👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**